client/python/datajunction/client.py

```diff
@@ -54,7 +54,7 @@ def list_metrics(self, namespace: Optional[str] = None) -> List[str]:
     def common_dimensions(
         self,
         metrics: List[str],
-    ):  # pragma: no cover # Tested in integration tests
+    ) -> List:  # pragma: no cover # Tested in integration tests
```

Comment:
Should this be `List[str]`?

Reply:
I thought so initially but then I realized we are returning dimension names with paths so this really is a `List[Dict]` right now.